### PR TITLE
Consistency edits 

### PR DESCRIPTION
**Pull request by afloresOSI**

### CODE REVIEW

File: content/add-organize-data/store-data/streams/streams-concept.md [hunk 1]
```diff
@@ -3,12 +3,13 @@ uid: ccStreams
...
+The SDS stream container uses metadata and tags to define information about the stream container itself. Metadata are key-value pairs that add context to the data, for example, the metadata key, *Manufacturer*, and a metadata value, *Caterpiller*. Tags are string values that represent stream attributes, for example, region. Organizing data using metadata tags and tags makes it much easier to retrieve data streams.
 
 ### <a name="streams-pi-server"></a>PI Server counterpart
 
 An SDS stream is comparable to a PI point in the PI Data Archive. For example, a float32 PI point might be sent to OSIsoft Cloud Services as an SDS stream with a type that contains a timestamp index and float32 value. If you use PI to OCS to import data into SDS, each PI point in the PI Data Archive will be created in SDS as individual stream and the data itself will be added as values in the stream.
+<!-- Angela Flores 6/10/21 - The last sentence still has two instances of passive voice. -->
```
Comment (rmuolic) [inline content/add-organize-data/store-data/streams/streams-concept.md:15]: Will the writer update or should we just update now? "will be" to "is".

File: content/add-organize-data/store-data/types/types-best-practices.md [hunk 2]
```diff
@@ -4,33 +4,35 @@ uid: bpTypes
 
 # Types best practices
 
-The following are best practices OSIsoft recommends for creating types and streams.
+OSIsoft recommends the following best practices for types and streams:
 
-- When you create SDS types, the most important consideration to be aware of is that types are immutable. Once created, additional properties or information cannot be added and existing properties cannot be deleted.
+- When you create SDS types, the most important thing to be aware of is that types are immutable. Once created, additional properties or information cannot be added and existing properties cannot be deleted.
```
Comment (rmuolic) [inline content/add-organize-data/store-data/types/types-best-practices.md:9]: Suggest to not use "thing". 
"When you create SDS types, it is important to be aware that types are immutable..."

File: content/overview/get-started/introduction-portal-interface.md [hunk 3]
```diff
@@ -2,46 +2,45 @@
...
 
 ![Feature menu](../images/feature-menu.png "Feature menu")
 
-Here is a short summary of each OCS feature:
+OCS includes the following features:
 
 - **Data Management**: Provides tools for setting up basic capabilities of OCS, including tenants, connections to PI Systems, the Sequential Data Store (SDS), and metadata rules for data streams.
 - **Data Collection**: Provides mechanisms to bring data from multiple sources and systems into a namespace within OCS.
-- **Visualization**: Allows you to view data trends and use assets to set up digital twins of real-world physical entities.
+- **Visualization**: Allows you to view data trends and use assets to set up digital twins of real-world physical entities. <!--Angela Flores 6/23/21 - this is the only place in the documentation that uses the phrase "digital twins of real-world physical entities". What feature is that trying to describe? The only thing in the Visualization portion of the documentation is Trend. -->
```
Comment (rmuolic) [inline content/overview/get-started/introduction-portal-interface.md:26]: "digital twins" sounds like an analogous term. I assume it means "simulated copies" or "simulations of real-world physical entities".

**Review by rmuolic — CHANGES_REQUESTED** [review]
Minor comments on my end.
I noticed comments addressed to the writer to clarify or add additional content. When/how will these comments be addressed?

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/pi-point-change.md [hunk 4]
```diff
@@ -4,18 +4,17 @@ uid: pi-point-change
...
-
-To see what types of point coercions are supported in PI Data Archive, please refer to the ["Allowable point type coercions"](https://docs.osisoft.com/bundle/pi-server/page/allowable-point-type-coercions.html) topic.
+* To maintain the data and continue streaming data for that particular tag, create a new SDS stream and copy the data to that new stream. Then, delete the SDS stream in question and restart the transfer.
+* To maintain the data, but not the streaming data for that particular tag, create a new transfer without the tag in question and start the transfer. 
+* To stream data for that particular tag, but not maintain previously transferred data, delete the SDS stream in question and restart the transfer.
 
+To see what types of point coercions are supported in PI Data Archive, please refer to the ["Allowable point type coercions"](https://docs.osisoft.com/bundle/pi-server/page/allowable-point-type-coercions.html) topic.<!--Angela Flores 6/28/21 should that be "coercions" or "conversions"? -->
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/pi-point-change.md:20]:
```suggestion
To see what types of point coercions are supported in PI Data Archive, refer to the ["Allowable point type coercions"](https://docs.osisoft.com/bundle/pi-server/page/allowable-point-type-coercions.html) topic.<!--Angela Flores 6/28/21 should that be "coercions" or "conversions"? -->
```

[the remark this answers is not in the document]
Comment (afloresOSI) [reply content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/pi-point-change.md]: It really is "coercions".

File: content/developer-guide/identity-dev/dev-guide-preconfigured-roles.md [hunk 5]
```diff
@@ -4,15 +4,11 @@ uid: Roles
 
 # Roles
 
-OSIsoft provides two preconfigured roles:
+OCS provides includes the following preconfigured roles:
```
Comment (afloresOSI) [inline content/developer-guide/identity-dev/dev-guide-preconfigured-roles.md:7]:
```suggestion
OCS includes the following preconfigured roles:
```

[the remark this answers is not in the document]
Comment (afloresOSI) [reply content/developer-guide/identity-dev/dev-guide-preconfigured-roles.md]: And this is why everyone needs their work doc QA'd.

File: content/add-organize-data/store-data/streams/streams-concept.md (hunk 1 above)
Comment (afloresOSI) [reply content/add-organize-data/store-data/streams/streams-concept.md]: It is still passive voice, but I can live with it.

**Review by rmuolic — APPROVED** [review]

**Pull request merged**